### PR TITLE
Remote 'kernelCommandListener' as it really listens to notebooks

### PR DESCRIPTION
**Pull request by rchiodo**
For #10152 

Another step in the removal of the 'notebook' references in kernels. The KernelCommandListener class was really listening to commands on notebooks so I just moved it.

### CODE REVIEW

File: src/notebooks/notebookCommandListener.ts [hunk 1]
```diff
@@ -45,6 +74,44 @@ export class NotebookCommandListener implements IDataScienceCommandListener {
         this.disposableRegistry.push(
             this.commandManager.registerCommand(Commands.NotebookEditorExpandAllCells, this.expandAll, this)
         );
+        this.disposableRegistry.push(
+            commandManager.registerCommand(
+                Commands.NotebookEditorInterruptKernel,
+                (context?: { notebookEditor: { notebookUri: Uri } } | Uri) => {
+                    if (context && 'notebookEditor' in context) {
+                        void this.interruptKernel(context?.notebookEditor?.notebookUri);
```
[the remark this answers is not in the document]
Comment (rchiodo) [reply src/notebooks/notebookCommandListener.ts:82]: Sure. This was already there, but I can fix it.

[the remark this answers is not in the document]
Comment (rchiodo) [reply src/notebooks/notebookCommandListener.ts:82]: We should add a linter rule.

[the remark this answers is not in the document]
Comment (rchiodo) [reply src/notebooks/notebookCommandListener.ts:82]: Or I can just search for 'void this' everywhere and fix it.

**Comment by rchiodo** [comment]
@DonJayamanne  did you have any more feedback?

File: src/notebooks/notebookCommandListener.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (DonJayamanne) [reply src/notebooks/notebookCommandListener.ts]: I've got a PR coming for that, will mention in standup.

**Comment by DonJayamanne** [comment]
Not sure why i didn't approve, sorry

**Review by DonJayamanne — APPROVED** [review]

**Pull request merged**